shared-data/command/schemas/7.json

```
@@ -460,6 +460,11 @@
           "title": "Pipetteid",
           "description": "Identifier of pipette to use for liquid handling.",
           "type": "string"
+        },
+        "pushOut": {
+          "title": "Pushout",
+          "description": "push the plunger a small amount farther than necessary for accurate low-volume dispensing",
+          "type": "number"
```

I think we also need this new argument in `JsonTranslator`. Currently, it's getting silently discarded from `dispense` and `dispenseInPlace` commands in JSON protocols.

---

Thank you Max! Ill add it!

---

added a fix for this in the embedded pr. https://github.com/Opentrons/opentrons/pull/13422